src/dec128.rs

```diff
@@ -786,6 +798,48 @@ impl d128 {
     pub fn is_zero(&self) -> bool {
         unsafe { decQuadIsZero(self) != 0 }
     }
+
```

Comment:
There is no from_le and to_le for floating point types. This is a floating point type. le/be for it is not well defined.